pkg/crypto/hash/hash_test.go

```diff
@@ -8,12 +8,12 @@ import (
 
 func TestHashAlgorithm_StringRoundtrip(t *testing.T) {
 	for algo := range Algorithms() {
-		if algo == Ed25519 {
-			// Ed25519 does hashing and signing in one time.
-			continue
-		}
 		str := algo.String()
 		hash1 := algo.CryptoHash()
+		if algo == Ed25519 || algo == None {
```

Comment:
This feels like it should move back up, above the `str` and `hash1` assignments?

Reply:
Fixed!